### PR TITLE
Grit pattern for trulens_eval migration

### PR DESCRIPTION
**Pull request by sfc-gh-jreini**
# Description

Create a [grit](https://about.grit.io/) pattern for migration from trulens_eval.

## Other details good to know for developers

Please include any other details of this change useful for _TruLens_ developers.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [ ] New Tests
- [ ] This change includes re-generated golden test results
- [ ] This change requires a documentation update

### CODE REVIEW

File: .grit/patterns/trulens_eval_migration.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Upgrade to TruLens v1.x
+
+This pattern upgrades code to TruLens v1.x by replacing deprecated imports and class usage.
+
+```grit
+engine marzano(0.1)
+language python
+
+any {
+  `Tru($session)` => `TruSession($session)`,
```
Comment (sfc-gh-chu) [inline .grit/patterns/trulens_eval_migration.md:15]: don't know anything about grit but I'm guessing it'll substitute imports(?). Can we add usage instructions to the migration guide?

Comment (sfc-gh-jreini) [reply .grit/patterns/trulens_eval_migration.md:15]: by imports do you mean args? And yeah I can add usage instructions. 

I'm also going to open up a PR to the grit standard lib once this is good to hopefully have it runnable by anyone with Grit with the command: 
`grit apply trulens_eval_migration`

See OpenAI's example: https://docs.grit.io/patterns/library/openai#change-openai-import-to-sync

Comment (sfc-gh-jreini) [reply .grit/patterns/trulens_eval_migration.md:15]: standard lib: https://github.com/getgrit/stdlib

**Review by sfc-gh-pmardziel — APPROVED** [review]

File: .grit/patterns/trulens_eval_migration.md (hunk 1 above)
Comment (morgante) [reply .grit/patterns/trulens_eval_migration.md:15]: FYI for the future you don't need to add patterns to the standard library to make them applyable. Just putting it in your repo here would make it runnable with `grit apply github.com/truera/trulens#trulens_eval_migration`.

**Pull request merged**